test/test_quality_gates_standalone.sh

```diff
@@ -16,6 +16,12 @@ function cleanup() {
 
   echo "Removing secret dynatrace-credentials-${PROJECT}"
   kubectl -n ${KEPTN_NAMESPACE} delete secret dynatrace-credentials-${PROJECT}
+
+  # print logs of dynatrace-sli-service
+  echo "Logs from: dynatrace-sli-service"
+  kubectl -n ${KEPTN_NAMESPACE} logs svc/dynatrace-sli-service -c dynatrace-sli-service
+  echo "Logs from: lighthouse-service"
+  kubectl -n ${KEPTN_NAMESPACE} logs svc/lighthouse-service -c lighthouse-service
```

Comment:
I know this has already been merged, but:
Does it make sense to retrieve logs AFTER we have deleted the dynatrace-sil-service deployment?

Reply:
indeed, this is fixed in https://github.com/keptn/keptn/pull/2993